src/wallet/db.cpp

```diff
@@ -136,7 +136,7 @@ bool IsSQLiteFile(const fs::path& path)
     }
 
     // Check the application id matches our network magic
-    return memcmp(Params().MessageStart(), app_id, 4) == 0;
+    return memcmp(Span{Params().MessageStart()}.data(), app_id, 4) == 0;
```

Comment:
nit: missing include, but do we need span? (+ in `sqlite.cpp`)

```suggestion
    return memcmp(Params().MessageStart().data(), app_id, 4) == 0;
```